### PR TITLE
Documentation for plone.staticresources

### PR DESCRIPTION
**Pull request by thet**
Currently, I've added a first draft of basic information.
TBC...

### CODE REVIEW

**Comment by ale-rt** [comment]
I anyway already merged this PR and left the branch open

**Comment by thet** [comment]
@ale-rt @polyester @pbauer @davilima6 @agitator 
please give me some hints about what to write and what's missing from these docs.

**Comment by polyester** [comment]
@thet it helps to state (in the first paragraph or so) quite clearly who the audience is of this documentation/addon. Is it important for: 

- site editors (I'm guessing not, but explicit is better than implicit)
- site admins who want to change settings for their Plone site, not necessarily related to theme/CSS things
- themers who want to change the look and feel of a site
- developers who want to customize a Plone site for their customers, and are confident with making a policy package or do other work on the command line, including maybe creating new patterns/widgets/mockup-thingies
- plone developers who want to help add/enhance this  add-on itself

and separate the document into seciions/headings according to those audiences.

File: README.rst [hunk 1]
```diff
@@ -5,6 +5,75 @@ plone.staticresources
...
+------------------------------------------------------
+
+The ``plone-compile-resources`` script can be used to compile bundles from the command line.
+In a glimpse, the script starts up a Plone instance, reads the resource- and bundle registrations from the registry and compiles a JS/CSS bundle out of that information.
+See ``plone-compile-resources --help`` for more information.
+
+When using the coredev buildout, the ``plone-compile-resources`` script is automatically generated.
+If you use a custom buildout, you might need to add something similar like::
+
+  [buildout]
```
Comment (stevepiercy) [inline README.rst]: Specify the syntax, do not use `::`.

```rst
like:

.. code-block:: ini
```

File: README.rst [hunk 2]
```diff
@@ -5,6 +5,75 @@ plone.staticresources
...
+  parts =
+    ...
+    zopepy
+  ...
+  [instance]
+  ...
```
Comment (stevepiercy) [inline README.rst]: To avoid warnings, use comments for the syntax. Instead of `...` use `# ...`.

File: README.rst [hunk 3]
```diff
@@ -5,6 +5,75 @@ plone.staticresources
...
+What has changed since Plone 5.1
+--------------------------------
+
+- All static resources - bundle resources, compiled bundles, external packages - from Products.CMFPlone.static has been moved here.
+- The bundle and resource registrations from the dependencies profile have been moved here.
+- The ``plone-compile-resources`` script has been moved here.
+- The toolbar pattern from Products.CMFPlone.static.toolbar has been moved to Mockup.
+- Mockup has been changed to use npm/yarn instead of bower.
+- The thememapper bundle from plone.app.theming has been moved here.
+- The resource editor bundle from plone.resourceeditor has been moved here.
```
Comment (stevepiercy) [inline README.rst]: In general, anything that is a filename, package name, and so on, should be enclosed with double backticks.  Example:

```rst
``plone.resourceeditor``
```

File: README.rst [hunk 4]
```diff
@@ -5,6 +5,75 @@ plone.staticresources
...
+For Patternslib, visit: http://github.com/patternslib/Patterns
+
+
+How to upgrade the resources in this package
+--------------------------------------------
+
+- Increase the npm package versions in package.json.
+- Run ``yarn upgrade``.
+- Run ``./bin/plone-compile-resources -b plone``, ``./bin/plone-compile-resources -b plone-logged-in`` and whatever bundle you're going to build in your buildout's root directory.
+  For more info on ``plone-compile-resources`` see: XXX
```
Comment (gforcada) [inline README.rst]: I guess this `XXX` is bound to be changed, right? :wink:

Comment (davilima6) [reply README.rst]: I believe so :) And used the same convention in a new commit. It's a WIP and your all the feedback has been very useful! Please continue so we fill the gaps.

File: README.rst (hunk 2 above)
Comment (davilima6) [reply README.rst]: I tried to implement all your suggestions, pls let me know if I got any wrong! thanks!

**Pull request merged**